cmake/modules/FindPlatinum.cmake

```diff
@@ -0,0 +1,75 @@
+#.rst:
+# FindPlatinum
+# --------
+# Finds the Platinum library
+#
+# This will define the following variables::
+#
+# PLATINUM_FOUND - system has Platinum
+# PLATINUM_INCLUDE_DIRS - the Platinum include directories
+# PLATINUM_LIBRARIES - the Platinum libraries
+# PLATINUM_DEFINITIONS - the platinum definitions
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_PLATINUM platinum QUIET)
+endif()
+
+find_library(PLATINUM_LIBRARY NAMES Platinum libPlatinum
+                              PATHS ${PC_PLATINUM_LIBDIR})
+
+find_path(PLATINUM_INCLUDE_DIR NAMES Platinum.h
+                               PATHS ${PC_PLATINUM_INCLUDEDIR}
+                               PATH_SUFFIXES Platinum)
+
+set(PLATINUM_VERSION ${PC_PLATINUM_VERSION})
+
+find_package(Neptune)
+
+if(ENABLE_INTERNAL_PLATINUM)
+  include(ExternalProject)
+
+  # Extract version
+  file(STRINGS ${CMAKE_SOURCE_DIR}/tools/depends/target/platinum/Makefile VER)
+  string(REGEX MATCH "VERSION=([^ ;]*)" PLATINUM_VER "${VER}")
+  set(PLATINUM_VER ${CMAKE_MATCH_1})
+
+  # allow user to override the download URL with a local tarball
+  # needed for offline build envs
+  if(PLATINUM_URL)
+    get_filename_component(PLATINUM_URL "${PLATINUM_URL}" ABSOLUTE)
+  else()
+    set(PLATINUM_URL https://github.com/lrusak/Platinum/archive/${PLATINUM_VER}.tar.gz)
```

Comment:
Should we fork to xbmc org?

Reply:
yes the plan would be to move them to be under the org